### DISCUSSION TITLE
Navigating raw text in quarto vscode with up and down arrows

### DISCUSSION THREAD

**Discussion by bs17smh** [quarto-dev/quarto-cli#3961] [Q&A] [answered]
Hello, 

I seem to remember in RMD files in R studio, that even though a section of raw text may register as one continuous line according to the script editor, you could still navigate up and down what you visually see as lines using the up and down arrows?

I am using quarto in vs code and if I press up or down in a long section of text it jumps to the beginning or end of the current section. 

I was wondering if there is any way around this? It seems simple but I can't seem to find any information on it, and it would be great to be able to edit without hopping to the mouse

**Comment by mcanouil** [comment]
I do not have the behaviour you described, so without knowing your settings, hard to tell you what to change or not.
I do not believe I changed anything regarding the cursor in my settings.

https://user-images.githubusercontent.com/8896044/212387309-fa062c41-c2ae-4682-be4d-6ec877cfef20.mov

**Comment by bs17smh** [comment]
Hi, thanks for the reply! 

However, I meant that this was an issue for me in raw text sections of qmd file. Its fine with small sections of text, but if i'm writing anything bigger I'd like to be able to hop around more? I understand if this just is what it is at the moment

I will have a dig in some of my settings
https://user-images.githubusercontent.com/64581667/212396557-14ed89c3-febf-4821-9456-d053a8fb3087.mp4

**Reply by mcanouil** [reply]
The behaviour of the cursor is the same for me everywhere.
It's not something set by the Quarto extension, so check your other extensions settings.

**Reply by bs17smh** [reply]
cool must be on my end then. I'll try and dig

thank you

**Reply by bs17smh** [reply]
This was because I had Vim loaded, all sorted thankyou

**Reply by mcanouil** [reply]
Can you mark one of the messages as the "answer" (or write a new one yourself to mark).
This way others will know it has been answered, even if by yourself.

**Comment by bs17smh** [comment, marked as answer]
This was because I had the vim extension loaded. You can change this by highlighting the text in question using shift v and then typing gq
More details here
https://stackoverflow.com/questions/43122175/automatically-hard-wrap-lines-at-column-in-vscode
